tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -47,11 +48,19 @@ def create_event(self, frames: Sequence[Mapping[str, str | bool]], platform: str
     def _get_trees_for_org(self, files: Sequence[str]) -> dict[str, RepoTree]:
         return {"test-org/repo": RepoTree(RepoAndBranch("test-org/repo", "master"), files)}
 
+    def _repo_tree_files(self, files: Sequence[str]) -> list[dict[str, Any]]:
+        return [{"path": file, "type": "blob"} for file in files]
+
+    def _repo_info(self) -> dict[str, str]:
+        return {"full_name": "test-org/repo", "default_branch": "master"}
+
     def _process_and_assert_code_mapping(
         self, files: Sequence[str], stack_root: str, source_root: str
     ) -> None:
         with (
-            patch(GET_TREES_FOR_ORG, return_value=self._get_trees_for_org(files)),
+            patch(f"{CLIENT}.get_tree", return_value=self._repo_tree_files(files)),
```

Comment:
After we call `get_tree` [1] we will call `filter_source_code_files` which reduces the number of files to store in the cache based on the extension.
https://github.com/getsentry/sentry/blob/c0f74327bce7cdb4f759ec4f76cd8a77068cea03/src/sentry/integrations/source_code_management/repo_trees.py#L198-L203